diff --git a/node_modules/react-native/scripts/react_native_pods_utils/script_phases.rb b/node_modules/react-native/scripts/react_native_pods_utils/script_phases.rb
--- a/node_modules/react-native/scripts/react_native_pods_utils/script_phases.rb
+++ b/node_modules/react-native/scripts/react_native_pods_utils/script_phases.rb
@@ -48,6 +48,6 @@ def get_script_phases_with_codegen_discovery(options)
         WITH_ENVIRONMENT="$RCT_SCRIPT_RN_DIR/scripts/xcode/with-environment.sh"
         /bin/sh -c "$WITH_ENVIRONMENT $SCRIPT_PHASES_SCRIPT"
         EOS
-    result = ERB.new(template, 0, '->').result(binding)
+    result = ERB.new(template, trim_mode: '->').result(binding)
     return result
 end
